internal/proxy/oauthproxy.go

```diff
@@ -1037,6 +1040,11 @@ func (p *OAuthProxy) Authenticate(rw http.ResponseWriter, req *http.Request) (er
 	}
 
 	req.Header.Set("X-Forwarded-User", session.User)
+
```

Comment:
can we also add some test in [oauthproxy_test.go](https://github.com/buzzfeed/sso/blob/master/internal/proxy/oauthproxy_test.go#L960) to test that the right header is being set?

Reply:
Sure, done!